### ISSUE TITLE
[Linux] All games give a black screen.

### ISSUE THREAD

**Issue by Sanaki** [libretro/quasi88-libretro#27]
Bisected, somewhat surprised by the result.
`bd96fe612dd8c0238b1f4d36e1803514142e322e is the first bad commit`

As stated, all games give a black screen after this commit. N mode shows the function key overlay at the bottom, but otherwise just black. Nothing even shows a cursor. No settings remedy this.

**Comment by bslenul** [comment]
Ah, I was looking if there was an issue about that, I just helped someone having the same issue on the SNES Mini :p 

He had `system/quasi88/disk.rom` and `system/DISK.ROM` (used by fMSX core), looks like Quasi88 prioritizes the uppercase, so it loads the file in `system/` and since it's meant for fMSX, well... :p

**Comment by Sanaki** [comment]
Ahh, I see. That should make it an easy fix then. I assume it scans for each file first in system/quasi88/ then system/, then moves onto the next. It just needs to be changed to look for -all- files in system/quasi88/ before falling back to system/, if it falls back there at all. To my knowledge, the fallback isn't documented anywhere and likely isn't being used by many users, if any.

**Comment by bslenul** [comment]
> To my knowledge, the fallback isn't documented anywhere and likely isn't being used by many users, if any.

Yeah, I was very surprised when I saw `[libretro INFO] [QUASI88]: Loaded /etc/libretro/system/DISK.ROM (0x00000800)` in his log file when all the other files were loaded from the "quasi88" folder, I had no idea it would be searching inside the main system folder as well oO

**Comment by negativeExponent** [comment]
similar to px68k (bios files in system/keropi), it probably makes more sense that it would just check on system/quasi88 and not on system/

**Comment by celerizer** [comment]
It would be trivial to have it only check the QUASI88 subfolder but I'm worried about breakage for existing users. Is this the best way to move forward?

Checking the downloads folder is an artifact of an earlier plan and should be removed either way.

**Comment by bslenul** [comment]
IMO the best way would be to check only in system/quasi88/, the docs and the info file only mention the "quasi88" subfolder anyway so I doubt many people put the files in system/ directly. But I guess scanning system/ is fine too as long as quasi88/ has complete priority (so search for both uppercase AND lowercase in that folder before switching to system/).
